src/ray/core_worker/common.h

```diff
@@ -168,4 +168,45 @@ struct PlacementGroupCreationOptions {
   const bool is_detached = false;
 };
 
+class ObjectLocation {
+ public:
+  ObjectLocation(NodeID primary_node_id, uint64_t object_size,
```

Comment:
I guess we can use const for all inputs?

Reply:
For parameters that are passed by value, my take is that const qualifier is an implementation detail and doesn't matter that much. But we can chat on this topic :)